internal/cmd/generate.go

```diff
@@ -141,7 +141,9 @@ func Generate(e Env, dir string, stderr io.Writer) (map[string]string, error) {
 		if sql.Gen.Go != nil {
 			name = combo.Go.Package
 		} else if sql.Gen.Kotlin != nil {
-			parseOpts.UsePositionalParameters = true
+			if sql.Engine == config.EnginePostgreSQL {
+				parseOpts.UsePositionalParameters = true
```

Comment:
This switch breaks MySQL since it's trying to replace non-existent `$1` to `?`. Not sure if some clever renaming would be useful here